cmd/homi/setup/cmd.go

```diff
@@ -342,12 +342,12 @@ func genServiceChainCommonGenesis(nodeAddrs, testAddrs []common.Address) *blockc
 				Reward: &params.RewardConfig{
 					MintingAmount: mintingAmount,
 					Ratio:         "100/0/0",
-					UseGiniCoeff:  false,
-					DeferredTxFee: false,
+					UseGiniCoeff:  true,
+					DeferredTxFee: true,
```

Comment:
@jeonghonah Are these two lines are okay for a service chain? It is for giving rewards to CN nodes, so it seems not fit for default settings for a service chain.